src/super_gradients/training/metrics/segmentation_metrics.py

```diff
@@ -174,27 +205,73 @@ def __init__(self, ignore_label=-100, dist_sync_on_step=False, metrics_args_prep
 
     def update(self, preds: torch.Tensor, target: torch.Tensor):
         predict, target = self.metrics_args_prep_fn(preds, target)
+        labeled_mask = self._handle_multiple_ignored_inds(target)
 
-        labeled_mask = target.ne(self.ignore_label)
         pixel_labeled = torch.sum(labeled_mask)
         pixel_correct = torch.sum((predict == target) * labeled_mask)
         self.total_correct += pixel_correct
         self.total_label += pixel_labeled
 
+    def _handle_multiple_ignored_inds(self, target):
+        if isinstance(self.ignore_label, list):
+            labeled_mask = None
+            for ignored_label in self.ignore_label:
+                if labeled_mask is None:
+                    labeled_mask = target.ne(ignored_label)
+                else:
+                    labeled_mask = torch.logical_and(labeled_mask, target.ne(ignored_label))
+        else:
+            labeled_mask = target.ne(self.ignore_label)
+
+        return labeled_mask
```

Comment:
Maybe name if `evaluated_classes_mask` ?

Reply:
sure, done in a9351123c4a0f04b854f552cf04291dacec9500f